salt/elife-dashboard/uwsgi.sls

```diff
@@ -28,25 +34,32 @@ app-uwsgi-conf:
         - require:
             - install-elife-dashboard
 
-old-uwsgi-app:
-    file.absent:
-        - name: /etc/init.d/uwsgi-app
-
-uwsgi-app:
+uwsgi-app-upstart:
     file.managed:
         - name: /etc/init/uwsgi-app.conf
         - source: salt://elife-dashboard/config/etc-init-uwsgi-app.conf
         - mode: 755
 
+{% if salt['grains.get']('osrelease') != "14.04" %}
+uwsgi-elife-dashboard.socket:
     service.running:
+        - enable: True
+        - require_in:
+            - uwsgi-app
+{% endif %}
+
+uwsgi-app:
+    service.running:
+        {% if salt['grains.get']('osrelease') != "14.04" %}
+        - name: uwsgi-elife-dashboard # newrelic is expecting 'uwsgi-app' (urgh)
```

Comment:
builder-private PR: https://github.com/elifesciences/builder-private/pull/296

because 14.04 isn't talking to newrelic right now it should be safe to merge. if it causes the build to fail I'll merge it today

Reply:
yeah, the uwsgi service is failing, merging.

Reply:
didn't help, bringing up a masterless instance to debug

Reply:
```python
Traceback (most recent call last):
  File "/srv/elife-dashboard/venv/lib/python2.7/site.py", line 773, in <module>
    main()
  File "/srv/elife-dashboard/venv/lib/python2.7/site.py", line 763, in main
    execsitecustomize()
  File "/srv/elife-dashboard/venv/lib/python2.7/site.py", line 553, in execsitecustomize
    import sitecustomize
  File "/srv/elife-dashboard/venv/local/lib/python2.7/site-packages/newrelic/bootstrap/sitecustomize.py", line 136, in <module>
    newrelic.agent.initialize(config_file, environment)
  File "/srv/elife-dashboard/venv/local/lib/python2.7/site-packages/newrelic/config.py", line 2548, in initialize
    log_file, log_level)
  File "/srv/elife-dashboard/venv/local/lib/python2.7/site-packages/newrelic/config.py", line 854, in _load_configuration
    'Unable to open configuration file %s.' % config_file)
newrelic.api.exceptions.ConfigurationError: Unable to open configuration file /srv/elife-dashboard/newrelic.ini.
```

investigating